readinglist/authentication.py

```diff
@@ -87,6 +87,7 @@ def permits(self, context, principals, permission):
         Authenticated users only are allowed.
         """
         PERMISSIONS = {
+            'batch': Authenticated,
```

Comment:
Why do we need a batch permission? Are all verbs authorized for the batch endpoint or only PATCH on the resource?